pom.xml

```diff
@@ -67,11 +67,6 @@
     <tag>HEAD</tag>
   </scm>
 
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
FWIW, this is bumping the baseline from 1.580.1 to 1.625.3, and JDK 7 instead of 6 previously. Intended?